recipes/pdb2pqr/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "pdb2pqr" %}
+{% set version = "3.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a6f5e2eefbd8bd2a072a89ee4026977de02dadd56e921095afac88d596b63820
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - pdb2pqr30=pdb2pqr.main:main
+    - dx2cube=pdb2pqr.main:dx_to_cube
+    - psize=pdb2pqr.psize:main
+    - inputgen=pdb2pqr.inputgen:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - mmcif_pdbx >=1.1.2
+    - pandas >=1.0
+    - pip
```

Comment:
Sorry, just noticed this: looks like `host` only needs the `python` and `pip` dependencies?

Reply:
Oh yes! Thanks for noticing that @dopplershift. I have changed it now.